### PR TITLE
Update does_sendgrid_offer_a_sandbox_or_test_environment.md

### PR DESCRIPTION
**Pull request by Zaftula**
Changed tense of trail period, added section for sandbox_mode parameter.

### CODE REVIEW

**Comment by mbernier** [comment]
What about sink?

**Comment by ksigler7** [comment]
hey @Zaftula can you please remove the information about the 30 day free trial? I realize that you didn't completely add it, but I think it's confusing in this file.

**Comment by ksigler7** [comment]
also, why two pull requests here? https://github.com/sendgrid/docs/pull/2613 Can you please combine them into one?

File: source/Classroom/Basics/API/does_sendgrid_offer_a_sandbox_or_test_environment.md [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
 layout: page
-zendesk_id: 203891358
 navigation:
   show: true
 ---
 
 We do not offer a true "sandbox" environment, however we do have a [free trial](https://sendgrid.com/transactional-email/pricing) which allows you to send 100 emails per day, and also gives you access to almost all of the features offered with our higher package levels. The free trial does not include access to a dedicated IP or IP whitelabeling.
 
-We also have a [sink email address]({{root_url}}/Classroom/Troubleshooting/Delivery_Issues/safely_test_your_sending_speed.html) that can help you test message throughput for your integration.
+If you are looking to test API integration, you can enable the sandbox_mode in the [mail_settings parameters of our Mail Send API](https://sendgrid.com/docs/Classroom/Send/v3_Mail_Send/sandbox_mode.html). **This will allow you to validate each parameter you include, and the structure of your JSON payload, all without actually sending a message.**
```
Comment (ksigler7) [inline source/Classroom/Basics/API/does_sendgrid_offer_a_sandbox_or_test_environment.md]: We merged in a pull request from an outside user that adds in the first part (which is why there's a conflict now). So when you combine the existing requests, you no longer need to update this section.

**Pull request closed without merging**